src/main/java/seedu/opus/logic/parser/DateTimeParser.java

```diff
@@ -34,7 +34,23 @@
             return Optional.empty();
         }
 
-        LocalDateTime output = LocalDateTime.ofInstant(date.toInstant(), ZoneId.systemDefault());
+        LocalDateTime output = convertDateToLocalDateTime(date);
         return Optional.ofNullable(output);
     }
+
+    public static Optional<DateGroup> parseDateGroup(String dateString) {
+        Parser parser = new Parser();
```

Comment:
This `Parser` is very confusing since we have our own `Parser` class as well. Maybe name this variable as `dateTimeParser` or something clearer?